app/helpers/submissions_helper.rb

```diff
@@ -86,7 +87,9 @@ def get_submissions_table_info(assignment, groupings)
           g[:grace_credits_used] = grouping.grace_period_deduction_single
           g[:section] = grouping.section
         end
-        g[:name_url] = get_grouping_name_url(grouping, result)
+        unless assignment.is_peer_review?
+          g[:name_url] = get_grouping_name_url(grouping, result)
+        end
```

Comment:
On second thought, should this just be a ternary statement with the line from the diff above @ line 62?